pkgs/development/misc/google-clasp/generate.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p nodePackages.node2nix
+
+set -eu -o pipefail
+
+exec node2nix --nodejs-10 \
+    -i node-packages.json -o node-packages.nix \
+    -c google-clasp.nix \
+     --no-copy-node-env -e ../../../development/node-packages/node-env.nix
```

Comment:
Two other occurrences:
- https://github.com/NixOS/nixpkgs/blob/0e296d5fcdaf32dd113e49738dfba6216c01a8da/pkgs/tools/package-management/nixui/generate.sh#L4
- https://github.com/NixOS/nixpkgs/blob/0e296d5fcdaf32dd113e49738dfba6216c01a8da/pkgs/tools/networking/airfield/deps.sh#L6

From my point of view, the less redundancy, the better.

Reply:
thanks for references. My thinking was that this creates non obvious coupling between what appears as unrelated modules. Anyway this is not really up to me to decide so unless someone more experienced comes in and gives definite answer I would keep it as is.

Reply:
IIRC we use a single node2nix in nixpkgs (hence the same node-env.nix) for everything, so this should be fine IMHO.
In the end we already have a lot of large diffs because of that, so I'd prefer to use the node-env.nix we already have **unless** we encounter issues that can be resolved by committing another one : )